client/reader/search-stream/blank.jsx

```diff
@@ -1,5 +1,27 @@
-import React from 'react';
+const React = require( 'react' );
 
-export default function BlankContent() {
-	return <p />;
-}
+const BlankContent = React.createClass( {
```

Comment:
You don't really need a full class here. Just a function that returns the right react tree.